### ISSUE TITLE
ERROR MailMessageBuilder  - Failed to send email

### ISSUE THREAD

**Issue by magdziarek** [biodiversitydata-se/ecodata#22]
I see this error too often -> needs investigation: 
2021-09-09 05:30:30,820 [pool-3-thread-17] ERROR mail.MailMessageBuilder  - Failed to send email
org.springframework.mail.MailSendException: Failed messages: com.sun.mail.smtp.SMTPSendFailedException: 530 5.7.0 Must issue a STARTTLS command first
; message exception details (1) are:
Failed message 1:
com.sun.mail.smtp.SMTPSendFailedException: 530 5.7.0 Must issue a STARTTLS command first

**Comment by mathieuLU** [comment]
Have we ever been able to send an email from ecodata ?
I know it works from BiCollect, since you send automatic messages to admin when surveys are submitted for example.

I compared the 2 config files (BC and Ecodata), they are the same. THe 2 EmailService.groovy are similar as well.

it really looks like a config pproblem, as if the port 587 was not correctly used

**Comment by magdziarek** [comment]
Yes, the email service works fine from BioCollect for the parts where we send email notifications to project admins...I think there might be error notifications sent also to the email address in config that don't work - like the notifications we used to get when there were problems with indexing with elasticsearch
